<launch>
    <node name="detect_grasps" pkg="gpd_ros" type="detect_grasps" output="screen">
        <param name="cloud_type" value="0" />
        <param name="cloud_topic" value="/gpd_input" />
        <param name="config_file" value="/home/ros_ws/src/grasp_generation/config/ros_eigen_params.cfg" />
        <param name="rviz_topic" value="plot_grasps" />
    </node>
 
        <!-- Launch the Python server script -->
    <!-- <node name="grasp_server" pkg="grasp_generation" type="main.py" output="screen"> -->
    <node name="grasp_server" pkg="grasp_generation" type="local_single_camera.py" output="screen">
        <!-- Adjust the path to your script if it's not installed -->
        <param name="use_sim_time" value="true" />
    </node>

    <!-- Launch RViz with the saved default configuration -->
    <!-- <node name="rviz" pkg="rviz" type="rviz" output="screen"/> -->
</launch>